dts/bindings/i2c/nordic,nrf-twim.yaml

```diff
@@ -6,3 +6,11 @@ description: Nordic nRF family TWIM (TWI master with EasyDMA)
 compatible: "nordic,nrf-twim"
 
 include: nordic,nrf-twi-common.yaml
+
+properties:
+    concat-buf-size:
+        type: int
+        required: false
+        description:
+            If concatenation buffer size is set, then multiple messages will be
+            concatenated into single transfers as long as there is space in buffer
```

Comment:
This description needs to be improved. It does not precise why the messages would be concatenated. And it is not true that the only criterion is the space in the buffer.